### DISCUSSION TITLE
Share your project with spotify-web-api-java

### DISCUSSION THREAD

**Discussion by alexeynefdef** [spotify-web-api-java/spotify-web-api-java#310] [Show and tell]
Hey everyone! I have an idea to share our current or finished projects witch use this lib.
It should resolve common problems and it's just nice to see different implementations.

**Comment by alexeynefdef** [comment]
So to start I'll share my small home project made mostly for myself and this app will help you to mix in key if you're DJ and you have your LPs as Spotify playlist. 
[Github repo](https://github.com/alexeynefdef/flowtherock-app)
Its live on server as well but currently in development so it might be not stable (currently working on frontend, I hate it).
Is it secure? Of course not.
You might already know that using Spotify API with multiple users in DEV mode is only possible by adding those manually,
so if you want to use it, let me know I'll grant a permission (it's up to 25 users)
[FlowTheRock](http://164.90.185.125)

**Comment by dargmuesli** [comment]
Reason for me to have a look into this project and ultimately taking over maintenance / doing a rewrite was the idea to **pull song metadata from Spotify and write it into local mp3 files!** After spending most of my time on getting this library to its current state my initial goal fell off radar for some time, but I got around programming **[spotitag](https://github.com/dargmuesli/spotitag)** a few months ago :rocket: 
Spotitag is written in Kotlin and a desktop application. Here is a ui preview:

![image](https://user-images.githubusercontent.com/4778485/197263651-6162d456-e8f1-45e8-b921-9782c46a3040.png)


I also created [spotilist](https://github.com/dargmuesli/spotilist) which helps to validate m3u playlists against Spotify.
